clients/ui/bff/api/app.go

```diff
@@ -16,9 +16,11 @@ const (
 	Version              = "1.0.0"
 	PathPrefix           = "/api/v1"
 	ModelRegistryId      = "model_registry_id"
+	RegisteredModelId    = "registered_model_id"
 	HealthCheckPath      = PathPrefix + "/healthcheck/"
 	ModelRegistry        = PathPrefix + "/model-registry/"
-	RegisteredModelsPath = ModelRegistry + ":" + ModelRegistryId + "/registered_models"
+	RegisteredModelsPath = ModelRegistry + ":" + ModelRegistryId + "/registered_models/"
+	RegisteredModelPath  = RegisteredModelsPath + ":" + RegisteredModelId
```

Comment:
```suggestion
	RegisteredModelPath  = RegisteredModelsPath + "/:" + RegisteredModelId
```
Without this change, we slightly change our API, requiring a trailing slash:

<img width="1441" alt="Screenshot 2024-09-04 at 4 29 53 PM" src="https://github.com/user-attachments/assets/d879fcc8-1416-4ba5-a287-662778211438">

@alexcreasy, what do you think? Should we keep the current behavior by default?

Reply:
Personally, I think it makes for an easier to use API if it functions with and without a trailing slash, rather than returning a 301. Some HTTP client libs I've used in the past, don't automatically follow the redirect - no idea if this is an issue we'd actually run into though :) 

This behaviour is perfectly valid within the HTTP spec, so I don't think this is a major issue?